examples/tpm/tpm_key_helpers.c

```diff
@@ -0,0 +1,361 @@
+/* helpers.c
```

Comment:
Please correct comment name. Please add `#ifdef WOLFSSH_TPM` around all this. Please remove and cleanup for functions that are not used.